### PR TITLE
doctl-apps: add page

### PR DESCRIPTION
**Pull request by cpetrella-sketch**
- [ ] The page (if new), does not already exist in the repository.
- [ ] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [ ] The page has 8 or fewer examples.
- [ ] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [ ] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [ ] The page description includes a link to documentation or a homepage (if applicable).

**Version of the command being documented (if known):**

### CODE REVIEW

**Review by mfrw — COMMENTED** [review]
@cpetrella-sketch Thank you for your contribution.
We prefer having a maximum of 8 examples per page.
Can we please prioritize and add only 8 examples :)

I would recommend going through the [guidelines](https://github.com/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines), if you haven't already :)

**Comment by CleanMachine1** [comment]
Could you please pick the most common/useful examples.

I am very sorry since I can see that you spent a considerable amount of time creating command examples.

File: pages/common/doctl-apps.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+- Create an app:
+
+`doctl apps create`
+
+- Create a deployment:
+
+`doctl apps create-deployment`
+
+- Deletes an app:
```
Comment (navarroaxel) [inline pages/common/doctl-apps.md:18]: Is this command interactive?
```suggestion
- Delete an app interactively:
```

File: pages/common/doctl-apps.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
+# doctl-apps
```
Comment (navarroaxel) [inline pages/common/doctl-apps.md:1]: The page title should match the command name:
```suggestion
# doctl apps
```

File: pages/common/doctl-apps.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+> Used to manage digitalocean apps.
+> More information: <https://docs.digitalocean.com/>.
+
+- Display commands for working with apps:
```
Comment (navarroaxel) [inline pages/common/doctl-apps.md:6]: Is the output of this command the help page?

Comment (cpetrella-sketch) [reply pages/common/doctl-apps.md:6]: I think so?

**Comment by cpetrella-sketch** [comment]
Oh, I thought I had 8 examples this time. Apparently I can't count. sorry about that.

File: pages/common/doctl-apps.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+
+`doctl apps create`
+
+- Create a deployment:
+
+`doctl apps create-deployment`
+
+- Delete an app interactively:
+
+`doctl apps delete`
```
Comment (sbrl) [inline pages/common/doctl-apps.md:16]: How does it know which app to delete? According to https://docs.digitalocean.com/reference/doctl/reference/apps/delete/, you need this:
```suggestion
`doctl apps delete {{app_id}}`
```

File: pages/common/doctl-apps.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
+# doctl apps
+
+> Used to manage digitalocean apps.
+> More information: <https://docs.digitalocean.com/>.
```
Comment (sbrl) [inline pages/common/doctl-apps.md:4]: Perhaps this link would be more specific?
```suggestion
> More information: <https://docs.digitalocean.com/reference/doctl/reference/apps>.
```

**Review by sbrl — COMMENTED** [review]
Thanks for the new page! I've left some comments below for you to review.

File: pages/common/doctl-apps.md [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+
+`doctl apps list-deployments`
+
+- Get logs:
+
+`doctl apps logs`
+
+- Update an app:
+
+`doctl apps update`
```
Comment (navarroaxel) [inline pages/common/doctl-apps.md:36]: The docs says the `--spec` argument is required. https://docs.digitalocean.com/reference/doctl/reference/apps/update/
```suggestion
`doctl apps update {{app_id}} --spec {{path/to/spec.yml}}`
```

File: pages/common/doctl-apps.md [hunk 7]
```diff
@@ -0,0 +1,36 @@
...
+
+`doctl apps list`
+
+- List all deployments:
+
+`doctl apps list-deployments`
+
+- Get logs:
+
+`doctl apps logs`
```
Comment (navarroaxel) [inline pages/common/doctl-apps.md:32]:
```suggestion
`doctl apps logs {{app_id}}`
```

File: pages/common/doctl-apps.md [hunk 8]
```diff
@@ -0,0 +1,36 @@
...
+
+- Update an app:
```
Comment (navarroaxel) [inline pages/common/doctl-apps.md:34]:
```suggestion
- Update a specific app with a given app spec:
```

File: pages/common/doctl-apps.md [hunk 9]
```diff
@@ -0,0 +1,36 @@
...
+
+- List all apps:
+
+`doctl apps list`
+
+- List all deployments:
+
+`doctl apps list-deployments`
+
+- Get logs:
```
Comment (navarroaxel) [inline pages/common/doctl-apps.md:30]:
```suggestion
- Get logs from a specific app:
```

File: pages/common/doctl-apps.md [hunk 10]
```diff
@@ -0,0 +1,36 @@
...
+
+- Get an app:
+
+`doctl apps get`
+
+- List all apps:
+
+`doctl apps list`
+
+- List all deployments:
```
Comment (navarroaxel) [inline pages/common/doctl-apps.md:26]:
```suggestion
- List all deployments from a specific app:
```

File: pages/common/doctl-apps.md [hunk 11]
```diff
@@ -0,0 +1,36 @@
...
+
+`doctl apps list-deployments`
```
Comment (navarroaxel) [inline pages/common/doctl-apps.md:28]: Based on docs the app id is required. https://docs.digitalocean.com/reference/doctl/reference/apps/list-deployments/
```suggestion
`doctl apps list-deployments {{app_id}}`
```

File: pages/common/doctl-apps.md [hunk 12]
```diff
@@ -0,0 +1,36 @@
+# doctl apps
+
+> Used to manage digitalocean apps.
+> More information: <https://docs.digitalocean.com/reference/doctl/reference/apps>.
+
+- Create an app:
+
+`doctl apps create`
+
+- Create a deployment:
```
Comment (navarroaxel) [inline pages/common/doctl-apps.md:10]:
```suggestion
- Create a deployment for a specific app:
```

File: pages/common/doctl-apps.md [hunk 13]
```diff
@@ -0,0 +1,36 @@
...
+
+`doctl apps create-deployment`
```
Comment (navarroaxel) [inline pages/common/doctl-apps.md:12]: The app id is required.
```suggestion
`doctl apps create-deployment {{app_id}}`
```

**Review by sbrl — APPROVED** [review]
Thanks, @cpetrella-sketch!

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**